huxley/www/js/components/TextInput.js

```diff
@@ -11,21 +11,36 @@ var cx = require('classnames');
 var TextInput = React.createClass({
```

Comment:
Let's add a docblock to this component describing why we're making them uncontrolled by-default, and linking to a task number.
```js
/**
 * TextInput is uncontrolled by default because blah blah...
 * <links to relevant task>
 */
var TextInput = React.createClass({...});
```